### ISSUE TITLE
Add tests for newly added theories

### ISSUE THREAD

**Issue by scarlehoff** [NNPDF/nnpdf#2017]
While it is too late for the ones we already have, we should check (as much as possible) that the theories are correct going forward.

The idea would be that theories here, when they have the new naming format (4xxxxxxx) are checked. Some possible tests, please suggest others:

- Check that the theory here is available in `theories_slim` with the same name and content.
- EW parameters are compatible

**Comment by felixhekhorn** [comment]
Question: we're talking about test within `nnpdf`, right? i.e. on FK tables?

then maybe some basic things first:
- check the thing is loadable
- check it yields a number when convoluted with NNPDF40_qed
- check it has a `theory` metadata key

> * EW parameters are compatible

now, this is a much more basic problem: do you really want to check theory compatibility? this has a non-trivial tail ... because it would extend to everything and that means backwards compatibility ... e.g. PTO, kThr, FNS, masses, etc.

same goes for the eko side: do we want to check that settings are compatible?

**Comment by scarlehoff** [comment]
> Question: we're talking about test within nnpdf, right? i.e. on FK tables?

No! Yes, within NNPDF but only on the XXX.yaml file. Downloading the theories in the CI and testing is not feasible so for that we just need to hope people is doing something sane.

> because it would extend to everything and that means backwards compatibility

I would only do it for new theories, just making sure that newly added theories don't have incompatible values.

**Comment by felixhekhorn** [comment]
do you want to add some very basic tests? á là 0 < mc <= mb <= mt ? Also, once upon a time we had a number of checks in yadism https://github.com/NNPDF/yadism/pull/240/files#diff-45326adfe602c7e7b10bfde28ef61cc33e469eff9ca46e69b5788762db3cd8f4 (some inherited from Apfel)
